### PR TITLE
Audit ES Translation of front-end-development-certificate.json

### LINKED ISSUE #6499: Translation to Spanish: Verify and update translation of front-end-development-certificate.json
- [x]  Claim Your Front End Development Certificate

### PR DESCRIPTION
**Pull request by justinian336**
Closes #6499

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/front-end-development-certificate.json [hunk 1]
```diff
@@ -240,12 +240,12 @@
       ],
       "type": "Waypoint",
       "challengeType": 7,
-      "nameEs": "Reclama tu certificado de desarrollo Front End",
+      "nameEs": "Reclama tu certificado de Desarrollo de interfaces",
```
Comment (FreakDroid) [inline seed/challenges/01-front-end-development-certification/front-end-development-certificate.json]: I don't know but in spanish is called "Desarrollo Front end", I think the name "Desarrollo de interfaces" Could be confused. BTW that should be "Reclama tu certificado de desarrollor Front End"

**Comment by FreakDroid** [comment]
I don't know but in spanish is called "Desarrollo Front end", I think the name "Desarrollo de interfaces" Could be confused. BTW that should be "Reclama tu certificado de desarrollor Front End"

**Comment by vtamara** [comment]
Thanks. Checking https://es.wikipedia.org/wiki/Front-end_y_back-end it is fine to translate front-end as interfaz.

**Comment by vtamara** [comment]
@bugron I checked this one.  It is a good translation.  Thank you @justinian336

**Comment by bugron** [comment]
@vtamara @justinian336 thanks, merged!

**Pull request merged**